inc/network/includes-network.php

```diff
@@ -0,0 +1,735 @@
+				}
+				?>
+				</td>
+			</tr>
+		<?php elseif ( ! allow_subdirectory_install() ) : ?>
+			<tr>
+				<th scope="row"><?php esc_html_e( 'Sub-domain Installation' ); ?></th>
+				<td>
+				<?php
+				_e( 'Because your installation is not new, the sites in your WordPress network must use sub-domains.' );
+					echo ' <strong>' . __( 'The main site in a sub-directory installation will need to use a modified permalink structure, potentially breaking existing links.' ) . '</strong>';
+				?>
+				</td>
+			</tr>
+		<?php endif; ?>
+		<?php if ( ! $is_www ) : ?>
+			<tr>
+				<th scope='row'><?php esc_html_e( 'Server Address' ); ?></th>
+				<td>
+					<?php
+					printf(
+						/* translators: %s: Host name. */
+						__( 'The internet address of your network will be %s.' ),
+						'<code>' . $hostname . '</code>'
+					);
+					?>
+				</td>
+			</tr>
+		<?php endif; ?>
+			<tr>
+				<th scope='row'><label for="sitename"><?php esc_html_e( 'Network Title' ); ?></label></th>
+				<td>
+					<input name='sitename' id='sitename' type='text' size='45' value='<?php echo esc_attr( $site_name ); ?>' />
+					<p class="description">
+						<?php _e( 'What would you like to call your network?' ); ?>
+					</p>
+				</td>
+			</tr>
+			<tr>
+				<th scope='row'><label for="email"><?php esc_html_e( 'Network Admin Email' ); ?></label></th>
+				<td>
+					<input name='email' id='email' type='text' size='45' value='<?php echo esc_attr( $admin_email ); ?>' />
+					<p class="description">
+						<?php _e( 'Your email address.' ); ?>
+					</p>
+				</td>
+			</tr>
+		</table>
+		<?php submit_button( __( 'Install' ), 'primary', 'submit' ); ?>
+	</form>
+	<?php
+}
+
+/**
+ * Prints step 2 for Network installation process.
+ *
+ * @since 3.0.0
+ *
+ * @global wpdb $wpdb     WordPress database abstraction object.
+ * @global bool $is_nginx Whether the server software is Nginx or something else.
+ *
+ * @param false|WP_Error $errors Optional. Error object. Default false.
+ */
+function network_step2( $errors = false ) {
+	global $wpdb, $is_nginx;
+
+	$hostname          = get_clean_basedomain();
+	$slashed_home      = trailingslashit( get_option( 'home' ) );
+	$base              = parse_url( $slashed_home, PHP_URL_PATH );
+	$document_root_fix = str_replace( '\\', '/', realpath( $_SERVER['DOCUMENT_ROOT'] ) );
+	$abspath_fix       = str_replace( '\\', '/', ABSPATH );
+	$home_path         = 0 === strpos( $abspath_fix, $document_root_fix ) ? $document_root_fix . $base : get_home_path();
+	$wp_siteurl_subdir = preg_replace( '#^' . preg_quote( $home_path, '#' ) . '#', '', $abspath_fix );
+	$rewrite_base      = ! empty( $wp_siteurl_subdir ) ? ltrim( trailingslashit( $wp_siteurl_subdir ), '/' ) : '';
+
+	$location_of_wp_config = $abspath_fix;
+	if ( ! file_exists( ABSPATH . 'wp-config.php' ) && file_exists( dirname( ABSPATH ) . '/wp-config.php' ) ) {
+		$location_of_wp_config = dirname( $abspath_fix );
+	}
+	$location_of_wp_config = trailingslashit( $location_of_wp_config );
```

Comment:
We actually want to make the changes in `config/application.php`

```suggestion
	$location_of_wp_config = trailingslashit( "/config" );
```

...although we could just replace `$location_of_wp_config` entirely with something new, since we're not talking about the `wp-config.php`